docusaurus/docs/migrating-from-toolkit.mdx

```diff
@@ -3,26 +3,24 @@ id: migrating-from-toolkit
 title: Migrating from Grafana Toolkit
 ---
 
-If you are using `@grafana/toolkit` to scaffold and develop plugins this guide will help you make the jump to plugin tools.
+If you are using `@grafana/toolkit` to scaffold and develop plugins, follow this guide to make the jump to our newest plugin tools.
 
 :::caution
-We [do not support](https://grafana.com/docs/grafana/latest/developers/angular_deprecation/) plugins written in Angular.
+Our plugin tools use ReactJS. We [do not support](https://grafana.com/docs/grafana/latest/developers/angular_deprecation/) plugins written in Angular.
 :::
 
-## Why migrate?
+## Why migrate to the `create-plugin` tool?
 
-- **More flexible:** With @grafana/create-plugin, developers have more control over their plugins, its dependencies, and can more easily customize the tooling to fit their specific needs.
-- **Faster development time:** With its out-of-the-box development environment @grafana/create-plugin can significantly reduce development time compared to using @grafana/toolkit.
-- **Improved testing capabilities:** Testing plugins with @grafana/create-plugin is much easier with github workflows that automate unit and e2e test runs whenever changes are pushed to github.
-- **Better documentation:** The documentation for @grafana/create-plugin is more comprehensive and easier to discover than @grafana/toolkit.
-
-:::warning
-Before running the following command we strongly suggest backing up the code. Ideally, store the plugin code in a Git repository and run the command on a clean branch to easily identify changes made by the migration command.
-:::
+- **More flexible:** With `create-plugin`, you have more control over your plugins and its dependencies. The additional control translates into greater ease of customizing the tooling to fit your specific needs.
+- **Faster development:** With its out-of-the-box development environment, `create-plugin` can significantly reduce development time compared to using `grafana-toolkit`.
+- **Improved testing capabilities:** Easily test plugins with `create-plugin` GitHub workflows that automate unit and e2e test runs whenever you push changes to GitHub. 
+- **Better documentation:** The documentation for `create-plugin` is more comprehensive and easier to discover than `grafana-toolkit`.
```

Comment:
Could you please add a new point here staying that the old tooling `grafana-toolkit` is deprecated and we won't offer support any longer?

Reply:
Fixed.